terraform/providers.tf

```
@@ -0,0 +1,13 @@
+terraform {
+  required_providers {
+    github = {
+      source  = "integrations/github"
+      version = "~> 4.0"
+    }
+  }
+}
+
+# Configure the GitHub Provider
+provider "github" {
+  token = ""
```

This should probably be something like:

```suggestion
  token = var.github_access_token
```